hack/lib/golang.sh

```diff
@@ -323,7 +323,7 @@ EOF
   go_version=($(go version))
   local minimum_go_version
   minimum_go_version=go1.9.1
-  if [[ "${go_version[2]}" < "${minimum_go_version}" && "${go_version[2]}" != "devel" ]]; then
+  if [[ "${minimum_go_version}" != $(echo -e "${minimum_go_version}\n${go_version[2]}" | sort -s -t. -k 1,1 -k 2,2n -k 3,3n | head -n1) && "${go_version[2]}" != "devel" ]]; then
```

Comment:
wow `sort -s -t. -k 1,1 -k 2,2n -k 3,3n | head -n1` :)

Reply:
sort=$(if which gsort &>/dev/null; then echo gsort; else echo sort)
.... | sort -V | head -n1

Reply:
Or even better, use `sort -V -C` that gives you rc 0 if the input was sorted.

Reply:
@sttts this is a backport for 1.9 of what's already fixed (merged) in master.

If you want to fix it differently I think the correct place it a PR to master branch.

Reply:
Alright, nvmd then.

Reply:
@sttts `sort -V -C` is a GNU extension, so that won't work for our macOS friends anyway. https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/sort.1.html

Reply:
We have dependencies on the brew gnu tools at many places. So gsort is available.